dotorg/dotorg-open-links-new-tab.js

```diff
@@ -0,0 +1,22 @@
+// ==UserScript==
+// @name         Opens all links in new tab
+// @namespace    http://tampermonkey.net/
+// @version      0.1
+// @description  This adds target="_blank" to all <a> tags
+// @author       You
+// @include        https://wordpress.org/support/plugin/the-events-calendar*
+// @include      https://wordpress.org/support/topic*
+// @include      https://wordpress.org/support/plugin/event-tickets*
+// @icon         https://www.google.com/s2/favicons?sz=64&domain=wordpress.org
+// @grant        none
+// ==/UserScript==
```

Comment:
Add `@downloadURL` and `@updateURL` so in case the script is updated it will be pulled automatically.

Reply:
Awesome.